client/src/components/TextInput/TextInput.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import {
+  InputContainer,
+  Input
+} from "./TextInput.styles";
+
+const TextInput = (props) => {
+  return (
+    <InputContainer className="search-bar__container">
+      <Input
+        type="search"
+        name="Search"
+        className="search-bar"
+        placeholder={props.placeholder}
+        dark={props.dark}

```

Comment:
I assume this is probably theme related? Is it possible to be a bit more descriptive so we know exactly what it is by just a glance?